ResearchKit/+Medable/ORKFormItemCell+Medable.m

```diff
@@ -0,0 +1,95 @@
+//
+//  ORKFormItemCell+Medable.m
+//  Axon
+//
+//  Created by me on 5/18/18.
+//  Copyright © 2018 Medable Inc. All rights reserved.
+//
+
+#import "ResearchKit.h"
+#import "ORKFormItemCell.h"
+
+@interface ORKFormItemCell ()
+- (void)ork_setAnswer:(id)answer;
+@end
+
+#pragma mark -
+
+@implementation ORKFormItemTextFieldBasedCell (Medable)
+
+- (void)ork_setAnswer:(id)answer
+{
+    [super ork_setAnswer:answer];
+    
+    @try
+    {
+        // in code below, use KVC, and performSelector:withObject:
+        // in order to bypass compiler/linker restrictions and
+        // avoid adding explicit dependencies to this module (RK)
+        
+        Class DBZxcvbn = NSClassFromString(@"DBZxcvbn");
+        NSString* answerFormatKeyPath = @"answerFormat.";
```

Comment:
since you have hardcoded values you can just go with @"answerFormat.isSecureTextEntry" and @"answerFormat.minimumPasswordStrength"

Reply:
added in https://github.com/Medable/ResearchKit/pull/3/commits/bc5d4f77882b610dfe0d0f256306470b18d86af1